src/components/AudioPage/audioPage.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './style.scss';
+import { MDBContainer, MDBRow, MDBCol } from 'mdbreact';
+import AudioPlayerPage from './audioPlayer';
+const AudioPage = () => {
+  const bookInfo = {
+    bookName: 'Mind without body',
+    bookCover: 'https://via.placeholder.com/353x498',
+    description: `is simply dummy text of the printing and typesetting industry.
+   Lorem Ipsum has been the industry's standard dummy text ever since
+   the 1500s, when an unknown printer took a galley of type and
+   scrambled it to make a type specimen book.galley of type and
+   scrambled it to make a type specimen book. .galley of type and
+   scrambled it to make a type specimen book.type specimen book`,
+  };
+
+  return (
+    <MDBContainer className="mt-5">
+      <MDBRow>
+        <MDBCol sm="5" size="12">
+          <div className="mt-5 d-flex align-items-center justify-content-center">
+            <img
+              src={bookInfo.bookCover}
+              width="353"
+              height="498"
+              alt="bookCover"
+            />
+          </div>
+        </MDBCol>
+        <MDBCol sm="7" size="12">
+          <div className="mt-5">
+            <h1 className="blue-text">{bookInfo.bookName}</h1>
+            <p className="mt-3 d-none d-sm-block text-2xl leading-7 text-justify">
+              {bookInfo.description}
+            </p>
+          </div>
```

Comment:
- you can break down this chunk of code into separate component and call it `bookInfo` and maybe you can use it here and in the book page

- also the audio page should be in `src/pages` folder and not in the component folder